staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -823,6 +823,23 @@ func (r *Request) transformResponse(resp *http.Response, req *http.Request) Resu
 	}
 }
 
+// truncateBody decides if the body should be truncated, based on the glog Verbosity.
+func truncateBody(body string) string {
+	max := 0
```

Comment:
How do I see the whole response?  v(10) should truncate nothing.

Reply:
Can you fix that in a follow up?  I was unable to debug an issue because I couldn't see the body.

Reply:
agreed, not sure how I missed that. Sending now (with test)